terraform/gen_teleport.yaml

```diff
@@ -83,13 +90,13 @@ defaults:
     "types.UserV2.SubKind": ""
 
     # Role
-    "types.RoleV3.Version": "v3"
-    "types.RoleV3.Kind": "role"
-    "types.RoleV3.SubKind": ""
-    "types.RoleV3.Spec.Options.MaxSessionTTL": "30h"
-    "types.RoleV3.Spec.Options.CertificateFormat": "standard"
-    "types.RoleV3.Spec.Allow.ReviewRequests.Where": ""
-    "types.RoleV3.Spec.Deny.ReviewRequests.Where": ""
+    "types.RoleV4.Version": "v4"
```

Comment:
IIUC this will cause existing terraform that used to emit a `v3` role to now emit a `v4` role if the user did not specify the version.  On its own, this would be a silently breaking change, since there is no change to role syntax, but behavior changes between `v3` and `v4`.  If anyone other than us is using this in prod already, the default value should probably be left as `v3` (@klizhentas may be able to comment on this).